addons/frag/functions/fnc_dev_addRound.sqf

```diff
@@ -0,0 +1,98 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Lambda.Tiger
+ * This function adds a round to be traced
+ *
+ * Arguments:
+ * 0: Projectile <OBJECT>
+ * 1: Add projectile event handlers <BOOL>
+ * 2: Is the round blue <BOOL>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [_proj, false, false] call ace_frag_dev_addRound;
+ *
+ * Public: No
+ */
+params [
+    "_proj",
+    ["_addEHs", true, [true]],
+    ["_sidePlayer", true, [true]]
+];
+
+// track round on each frame
+// Create entry in position array from hashmap
+if (_sidePlayer) then
+{
+    GVAR(dev_trackLines) set [getObjectID _proj, [1, [getposATL _proj], [0, 0, 1, 1]]];
+} else
+{
+    GVAR(dev_trackLines) set [getObjectID _proj, [1, [getposATL _proj], [1, 0, 0, 1]]];
+};
+// eventhandler to track round and cleanup when round is "dead"
+[
+    {
+        if (isGamePaused) exitWith {};
+        params ["_par", "_h"];
+        _par params ["_proj"];
```

Comment:
```suggestion
        _par params ["_projectile"];
```

More [Coding guidelines](https://ace3.acemod.org/wiki/development/coding-guidelines#112-private-variable-naming) stuff.

Reply:
Expanded everything to be the as verbose as it can be while still being concise.